lib/web/apiserver_test.go

```diff
@@ -2001,6 +2003,7 @@ func TestClusterDatabasesGet(t *testing.T) {
 	resp = testResponse{}
 	require.NoError(t, json.Unmarshal(re.Bytes(), &resp))
 	require.Len(t, resp.Items, 1)
+	require.Equal(t, resp.TotalCount, 1)
```

Comment:
Same comment as above. And this occurs a few more times below. I would just take a pass and check all new assertions.